app/controllers/surveys_controller.rb

```diff
@@ -13,6 +13,7 @@ def send_survey
       return render json: {error: :survey_not_found}, status: :not_found
     end
 
+    @survey.generate_models
```

Comment:
Yeah, maybe if the question has a `first_question`. 

We can change it later if we need more strict requirements. This check may not be valid if we generate the models but they aren't valid, so we'd have to rollback the generation of those models. We can think about it later.